packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -63,65 +61,26 @@ export enum NotificationBannerCTATypes {
   remote_notification_cta = 'remote_notification_cta',
 }
 
-interface StateProps {
-  backupCompleted: boolean
-  numberVerified: boolean
-  goldEducationCompleted: boolean
-  dismissedInviteFriends: boolean
-  dismissedGetVerified: boolean
-  verificationPossible: boolean
-  dismissedGoldEducation: boolean
-  incomingPaymentRequests: PaymentRequest[]
-  outgoingPaymentRequests: PaymentRequest[]
-  extraNotifications: IdToNotification
-  reclaimableEscrowPayments: EscrowedPayment[]
-  rewardsEnabled: boolean
-}
-
-interface DispatchProps {
-  dismissNotification: typeof dismissNotification
-  dismissInviteFriends: typeof dismissInviteFriends
-  dismissGetVerified: typeof dismissGetVerified
-  dismissGoldEducation: typeof dismissGoldEducation
-  openUrl: typeof openUrlAction
-}
+function NotificationBox() {
+  const [currentIndex, setCurrentIndex] = useState(0)
+  const backupCompleted = useSelector((state) => state.account.backupCompleted)
```

Comment:
Nit: Maybe worth pulling all of `state.account` then destructuring to get the properties you need? Looks like that will save a few lines